packages/hooks-react/src/series/useNextEpisode.ts

```diff
@@ -3,14 +3,19 @@ import type { Series } from '@jwp/ott-common/types/series';
 import ApiService from '@jwp/ott-common/src/services/ApiService';
 import { getModule } from '@jwp/ott-common/src/modules/container';
 import { CACHE_TIME, STALE_TIME } from '@jwp/ott-common/src/constants';
+import { useTranslation } from 'react-i18next';
 
 export const useNextEpisode = ({ series, episodeId }: { series: Series | undefined; episodeId: string | undefined }) => {
   const apiService = getModule(ApiService);
 
+  // Determine currently selected language
+  const { i18n } = useTranslation('menu');
+  const language = i18n.language;
+
   const { isLoading, data } = useQuery(
-    ['next-episode', series?.series_id, episodeId],
+    ['next-episode', series?.series_id, episodeId, language],
     async () => {
-      const item = await apiService.getEpisodes({ seriesId: series?.series_id, pageLimit: 1, afterId: episodeId });
+      const item = await apiService.getEpisodes({ seriesId: series?.series_id, pageLimit: 1, afterId: episodeId, language });
```

Comment:
Is the `'menu'` namespace given to `useTranslation` accidental or on purpose? I think we can remove it.

Alternatively, saving some lines, you can do this as well:

```tsx
import { getI18n } from 'react-i18next';

const item = await apiService.getEpisodes({ seriesId: series?.series_id, pageLimit: 1, afterId: episodeId, language: getI18n().language });
```

Reply:
Forget the getI18n suggestion. That doesn't make it reactive to language changes